JohnsonCountyJail.yaml

```diff
@@ -882,428 +886,374 @@ entityDefinitions:
-    fqn: "ol.subjectof"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "JohnsonCounty_IowaCity_JCBooking"
-    entitySetName: "JohnsonCounty_IowaCity_JCSubjectOf"
-    propertyDefinitions:
-      ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCSubjectOf"
-  JohnsonCounty_IowaCity_JCSubjectOf2:
-    fqn: "ol.subjectof"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "JohnsonCounty_IowaCity_JCBond"
-    entitySetName: "JohnsonCounty_IowaCity_JCSubjectOf"
-    propertyDefinitions:
-      ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCSubjectOf2"
-  JohnsonCounty_IowaCity_JCChargedWith:
-    fqn: "ol.chargedwith"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "JohnsonCounty_IowaCity_JCOffenses"
-    entitySetName: "JohnsonCounty_IowaCity_JCChargedWith"
-    propertyDefinitions:
-      ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCChargedWith"
   JohnsonCounty_IowaCity_JCSentencedWith1:
     fqn: "ol.sentencedwith"
+    entitySetName: "JohnsonCounty_IowaCity_JCSentencedWith"
     src: "JohnsonCounty_IowaCity_JCInmates"
     dst: "JohnsonCounty_IowaCity_JCSentences"
-    entitySetName: "JohnsonCounty_IowaCity_JCSentencedWith"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
+      general.datetime:
+        type: "general.datetime"
+        column: "Date_In"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["M/d/yyyy", "d-MMM-yy", "MM/dd/yyy", "M/dd/yyyy", "MM/d/yyyy", "dd-MMM-yy", "yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/Chicago"
     name: "JohnsonCounty_IowaCity_JCSentencedWith1"
-  JohnsonCounty_IowaCity_JCSentencedWith2:
-    fqn: "ol.sentencedwith"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "Sentences2"
-    entitySetName: "JohnsonCounty_IowaCity_JCSentencedWith"
-    propertyDefinitions:
-      ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCSentencedWith2"
-  JohnsonCounty_IowaCity_JCSubjectOf:
-    fqn: "ol.subjectof"
+
+  hasrelationshipwith:
+    fqn: "ol.hasrelationshipwith"
+    entitySetName: "JohnsonCounty_IowaCity_JCHasRelationshipWith"
     src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "JohnsonCounty_IowaCity_JCJailStay"
-    entitySetName: "JohnsonCounty_IowaCity_JCSubjectOf"
+    dst: "JohnsonCounty_IowaCity_BondRelative"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCSubjectOf"
-  JohnsonCounty_IowaCity_JCOfLength:
-    fqn: "ol.oflength"
-    dst: "JohnsonCounty_IowaCity_JCJailStayLength"
+        type: "ol.id"
+        column: "Person_Post"
+      ol.role:
+        type: "ol.role"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Kin"
```

Comment:
in "role" shouldn't we instead write in the relative value in the Person_Post column? e.g., "mother". If it says things like that also we should prob revers src and dst entity sets.